lib/repository/forum/ForumDelenMeldingRepository.php

```diff
@@ -118,20 +124,14 @@ private function stuurMelding($ontvanger, $auteur, $post, $draad, $deel, $berich
 		);
 
 		// Stel huidig UID in op ontvanger om te voorkomen dat ontvanger privé of andere persoonlijke info te zien krijgt
-		ContainerFacade::getContainer()->get(SuService::class)->overrideUid($ontvanger->uid);
-
-		// Verzend mail
-		try {
+		$this->suService->alsLid($ontvanger->account, function () use ($draad, $deel, $ontvanger, $values, $bericht) {
```

Comment:
Ik ben bang dat `$ontvanger->account` hier `null`  kan zijn als het account van iemand die een forumdeel volgt verwijderd is.